services/compileGRPCStubs/index.ts

```diff
@@ -118,6 +118,36 @@ const zippedServiceFilePath = (orgId: string, fileName: string): string => {
   return `${directory}/${fileName}`;
 };
 
+const findFile = (files: string[], pattern: string): string => {
+  for (const file of files) {
+    if (file.includes(pattern)) {
```

Comment:
can you `Array.find` instead of `for....of`

Reply:
Updated in the latest commit